File: Mathematical/Java/Find_the_median.java
class Solution
{
    public int find_median(int[] v)
    {
        // Code here
        Arrays.sort(v);
        int n = v.length;
        int start = 0;
        int mid = start + (n-start)/2;
        if (n%2==0) {
            int avg = (int) (v[mid] + v[mid-1])/2;
            return avg;
        }
        return v[mid];
    }
}

File: Solutions/Python/Find_the_median.py
class Solution:
	def find_median(self, v):
		# Code here
		v.sort()
        n = len(v)
        start = 0
        mid = start + (n-start)//2
        if n % 2 == 0: 
            avg = int((v[mid] + v[mid-1])/2)
            return avg
        return v[mid]
